plugins/nic/src/plugin.cpp

```diff
@@ -324,6 +329,13 @@ class plugin final : public virtual loader_plugin<nic_loader>,
     return {.source = true};
   }
 
+  auto make(invocation inv, session ctx) const
+    -> failure_or<operator_ptr> override {
+    argument_parser2::operator_("nics").parse(inv, ctx).ignore();
+    // XXX: Figure out what to do with nic_loader
```

Comment:
I think that's for a follow-up, just like every other loader, saver, parser, or printer.